### ISSUE TITLE
Go back

### ISSUE THREAD

**Issue by ManUtopiK** [nuxt/nuxt#27084]
It should be nice to have a go back button `window.history.back()` here :
https://github.com/nuxt/assets/blob/aa803dceebb99a83b65401bcf90c259bc0d63a62/packages/templates/templates/error-404/index.html#LL71C9-L71C10

**Comment by MirkoJa** [comment]
The link in this issue leads to https://github.com/nuxt/ui-templates which is archived since 2024. @danielroe this can be closed.

**Comment by danielroe** [comment]
nuxtathon closed @MirkoJa

**Comment by OrbisK** [comment]
> The link in this issue leads to https://github.com/nuxt/ui-templates which is archived since 2024. [@danielroe](https://github.com/danielroe) this can be closed.

Thats correct, but I was referring to the 404 error pages, which still exist.

**Comment by Norbiros** [comment]
I agree that this should be further discussed.

It won't hurt anybody to add this as a new feature, but I don't see the reason why you need that. It is a developer-only error page, you can just use your browser controls to go back, refresh and do way more, like hard refresh, go forward etc.

**Comment by MirkoJa** [comment]
I think this needs to be case-dependent for a good ux then, if we really want this.
- if you come from an external page and type in a url that leads to the error-page: href="/"
- if there is no history: href="/"
- if the last history entry is NOT an external page: window.history.back()

Otherwise the plain window.history.back() solution would be worse in my eyes.
